Add Ocean's Logger and use it

Geod24 opened the pull request:
Nothing to see here

Geod24 committed:
Replace vibe.core.log with Ocean's logger

Geod24 committed:
Remove quorum set logInfo

AndrejMitrovic commented on source/agora/utils/Log.d:
```diff
@@ -0,0 +1,37 @@
+/// Initialize the logger
+static this ()
+{
+    import ocean.util.log.Logger;
```
This import already exist in module scope.

AndrejMitrovic commented on source/ocean/util/log/Logger.d:
```diff
@@ -782,7 +782,7 @@ public final class Logger : ILogger
     ***************************************************************************/
 
-    public Logger append (Level level, lazy cstring exp)
+    public Logger append (Level level, lazy cstring exp) @trusted
```
Commit message says nothrow but I don't see it here.

AndrejMitrovic commented:
Ok just kill the BanManager commit and it will go green?

Geod24 commented:
> Ok just kill the BanManager commit and it will go green?

I had to remove an unused function because the C symbol it references does not exist on Musl...
And one allocation test fails, let me look into it.

Geod24 committed:
Add a stripped version of ocean as submodule

This version is mostly of interest because it has Formatter and Logger.
Many modules have been removed, as Ocean is not portable accross POSIX platforms.

Geod24 commented:
Updated, now it's less than 200 LoC!

AndrejMitrovic commented on source/agora/consensus/data/UTXOSet.d:
```diff
@@ -188,15 +189,15 @@ private class UTXODB
         const db_exists = utxo_db_path.exists;
         if (db_exists)
-            logInfo("Loading UTXO database from: %s", utxo_db_path);
+            log.info("Loading UTXO database from: %s", utxo_db_path);
```
`%s` => `{}`

AndrejMitrovic commented on source/agora/consensus/data/UTXOSet.d:
```diff
@@ -188,15 +189,15 @@ private class UTXODB
 
         if (db_exists)
-            logInfo("Loaded database from: %s", utxo_db_path);
+            log.info("Loaded database from: %s", utxo_db_path);
```
`%s` => `{}`
